### PR TITLE
adding analytics v2 button to admin

### PR DESCRIPTION
**Pull request by tvweinstock**

### CODE REVIEW

File: _dashboard/src/App.vue [hunk 1]
```diff
@@ -117,51 +127,62 @@ import 'regenerator-runtime/runtime';
...
-      get('/admin/worker').then(response => {
-        const jobIDs = response.data.job_ids;
-
-        if (jobIDs.length === 0) {
-          // User already registered
-          this.loadData(indicesResponse.data);
-        } else {
-          // New user -> wait for indices to be generated
-          const notifId = store.pushNotification('loading', 'Your indices are being generated, it may take a few minutes...');
+    get('/admin/indices')
```
Comment (therealwebby) [inline _dashboard/src/App.vue]: is this just prettier reformatting?

File: _dashboard/src/App.vue [hunk 2]
```diff
@@ -59,10 +59,20 @@
...
       </div>
-      <div class="has-text-centered">
-        <a class="side-menu-view-dashboard" :href="'https://www.algolia.com/apps/932LAAGOT3/explorer/browse/' + encodeURIComponent(user.email + '#' + $route.params.index) + '?_pid=81919'" target='_blank'>
-          <span>VIEW INDEX IN DASHBOARD</span>
-        </a>
+      <div class="tile is-ancestor">
+        <div class="tile is-parent is-vertical columns is-vcentered">
+          <div class="tile has-text-centered">
+            <a class="side-menu-view-dashboard"
+            :href="'https://www.algolia.com/apps/932LAAGOT3/explorer/browse/' + encodeURIComponent(user.email + '#' + $route.params.index) + '?_pid=34641'" target='_blank'>
```
Comment (therealwebby) [inline _dashboard/src/App.vue]: `pid` ftw 👍

File: _dashboard/src/App.vue (hunk 1 above)
Comment (tvweinstock) [reply _dashboard/src/App.vue]: indeed it is! what do you think?

Comment (therealwebby) [reply _dashboard/src/App.vue]: 👍

**Pull request merged**